Week3/app/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,49 @@
+<resources>
+    <!-- Base application theme. -->
+    <style name="AppTheme" parent="Theme.MaterialComponents.Light.DarkActionBar">
+        <!-- Customize your theme here. -->
+        <item name="colorPrimary">@color/colorPrimary</item>
+        <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
+        <item name="colorAccent">@color/colorAccent</item>
+        <item name="fontFamily">@font/bitter_bold</item>
+    </style>
+
+    <!-- Style for the image in the header. -->
+    <style name="imageHeaderSize" >
+        <item name="android:layout_height">@dimen/header_image_size</item>
+        <item name="android:layout_width">@dimen/header_image_size</item>
+    </style>
+
+    <!-- Style for views with wrap_content for both height and width. -->
+    <style name="wrapperSize">
+        <item name="android:layout_height">wrap_content</item>
+        <item name="android:layout_width">wrap_content</item>
+    </style>
+
+    <!-- Style for views with match_parent for both height and width. -->
+    <style name="matchParentSize">
+        <item name="android:layout_height">match_parent</item>
+        <item name="android:layout_width">match_parent</item>
+    </style>
+
+    <style name="titleTextStyle" parent="wrapperSize">
```

Comment:
This doesn't make sense, you are inheriting from wrapperSize which defines only width and height, and then you define some text properties in the new style. 
If you are creating text styles you almost always want to inherit from one of the `TextAppearance` styles